docs/index.rst

```diff
@@ -94,6 +94,7 @@ Files, I/O, and Communication
    io/votable/index
    io/misc
    samp/index
+   asdf
```

Comment:
Maybe move this to be subpage of `io/misc` to be consistent with the subpackage that it is replacing?

Reply:
(reverted all documentation changes)